core/README.md

```diff
@@ -177,6 +177,94 @@ machine.start(object).then(({object}) => {
 });
 ```
 
+## Object Workflow History
+
+Machine configuration
+```
+var machine = new Machine({history, ...});
+```
+**history** is a DAO that provides a possibility to create and read object workflow history records. You can find its API (and DB specific implementation) [here](https://github.com/OpusCapita/fsm-workflow/tree/master/history).
+
+Machine writes history records for all object transitions within the workflow.
+It happens when you start workflow
+```
+machine.start({object, user, description})
+```
+or you send an event
+```
+machine.sendEvent({ object, event, user, description})
+```
+In both cases, new history records are created.
+Here
+- **object** (required) - business object of the following structure
+```javascript
+{
+  businessObjId,
```

Comment:
Add example about getters - not many guys are aware about it.

Reply:
In JS there is no effort to add getter or property to an object.
See https://javascriptplayground.com/es5-getters-setters/